### DISCUSSION TITLE
[UserStoriesToRefine] #4 As <user>  I want a website where users can easily leave their contact information (email and/ or WhatsApp)

### DISCUSSION THREAD

**Discussion by emecas** [emecas/civic-tech-website-consultancy-pilot-translation#25] [General]
### As <user>  I want a website where users can easily leave their contact information (email and/ or WhatsApp)

**Comment by emecas** [comment]
## As a [USER NAME] I can [ACHIEVE SOMETHING], so that [VALUE STATEMENT]

## Level 2- As a cacao producer I want to provide my email as part of my contact  information, so that I can be contacted.

## Level 2- As a cacao producer I want to provide my WhatsApp as part of my contact information, so that I can be contacted.

**Reply by UnitedByChocolate** [reply]
Acceptance Criteria: 
1. Have an opt-in button to opt into a United by Chocolate WhatsApp group
2. Have an email opt-in at the bottom of the landing/ homepage
